src/internal/m365/collection/drive/collections_tree_test.go

```diff
@@ -635,6 +647,51 @@ func (suite *CollectionsTreeUnitSuite) TestCollections_TurnTreeIntoCollections()
 					fileID()),
 			},
 		},
+		{
+			name:           "all folders not moved - path separator string check",
```

Comment:
nit: for this and other tests that were updated could you please update the names so they more closely match what's happening? It's not quite clear now what things are not moved because they're not mentioned and what things are not moved because the "new" path in the tree matches the old path

Reply:
hmm, nothing should be moved here.  I don't know how to make that more obvious.  Suggestions?

Reply:
hmm maybe `all tree folders not moved all nontree folders not moved`? It's getting a bit wordy though

Reply:
hmm, how about: `nothing in the tree was moved but there were some folders in the previous paths that didn't appear in the delta so those have to appear in the new previous paths but those weren't moved either so everything should have the same path at the end`